### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Strangeal**
### Capstone Project

So far it was enjoyable😊 implementing all the necessary requirements for the project I learned so many things and I have implemented so many things which I didn't know before, what I did in my whole project.

- Set up the project with GitHub flow and linters.
- Added the HTML skeletons and made javascript and stylesheet files.
- Added images folder
- Used CSS for styling my project.
- Added the interactions for mobile and desktop versions.
- Successfully debugged the linters error.
- Dynamically render the featured speaker section
- I used JavaScript with best practices 

Finally, I updated the README.md file and also added the screenshots and link of my recoreded video

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,101 @@
...
\ No newline at end of file
+# shin_capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+
+> This is a module 1: week 5 project in Microverse full stack course.
+
+
```
Comment (BertrandConxy) [inline README.md]: - [x] Please describe very well what your website is about here in the project description section. From what I have seen, I would suggest you say like 'The Clickbank website is a platform that hosts affiliate marketing conferences and events. You can browse the featured speakers...' so, please describe the project very well in this section 😃 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+## Built With
+
+- html
+- css
+- Javascript
+
+
+## Live Demo
+
+[**See Project 🚀**](https://strangeal.github.io/shin_capstone/)
```
Comment (BertrandConxy) [inline README.md]: - [x] Your deployed project link is not working and I really don't know the reason why 😞. This is what I can see when I checkout the link 👇 
image

I would suggest that you deploy it again and make sure that the link is working very well 😉

File: README.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Inspiration
+- etc
+
```
Comment (BertrandConxy) [inline README.md]: - [x] In the project requirements, you are expected to give credit and  Acknowledgements to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
Please add the credit here in the acknowledgments section 👍

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Strangeal ✋, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 

## Highlights 💪 

- Great use of comments for code explanation
- No linter errors
- correct  GitHub flow is used
- good PR
- Awesome UI 💯 

## Required Changes ♻️

- [x] Your video presentation was great 👏 However **you forgot to tackle one important part which is the piece of code that you are proud of.** It is important to note that **this showcases your ability to explain technical concepts which is a great skill to possess.** I would suggest you record another video and include that part too 😉 
⚠️ **REMINDER:** It is recommended to use the [Loom platform](https://www.loom.com/) to record your video presentation 😎  

_Check the comments under the review._

## Optional suggestions

- [x] [OPTIONAL] Your project is awesome 😮 👏 💯 . In addition to what you have done, I think it would be better **if you give this GitHub repo the same as the website** for example `clickbank website` or just `clickbank`. By doing this, It will sound more specific and professional 😃 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @Strangeal,

# STATUS APPROVED 👏🏽  👏🏽 👏🏽 
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉

**[OPTIONAL]** When the user stays in the about section and clicks on the logo it should back to the home page, kindly make sure that the logo works as expected, for a better user experience. 😄


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**